values/wire-server/demo-secrets.example.yaml

```diff
@@ -4,8 +4,8 @@ brig:
     zAuth:
       # generate zauth public/private keys with the 'zauth' executable from wire-server:
       # ./dist/zauth -m gen-keypair -i 1
-      privateKeys:
-      publicKeys:
+      publicKeys: "<public key>"
+      privateKeys: "<private key>"
```

Comment:
@ChrisPenner  question: does brig fail to start if the key contains an invalid key? Or does it only lead to a 500 when attempting to get an access token?
Same question for nginz.

I'd like to prevent the helm install from succeeding if the config is wrong - that's why initially these were left empty. Or do you have another idea on how to fail earlier when the helm config is wrong?
(Minor nitpick: it's `keys` in plural as newline separated keys are accepted)

Reply:
Ahh; I had assumed it would fail on startup but didn't try testing it, I added a key here because when going through the tutorial I was very confused as to whether this was supposed to be an array or a string; the plural name implies an array; the string here hints at the correct format. We could perhaps use a comment to indicate this instead?